index.html

```diff
@@ -366,7 +366,76 @@ <h4>Card with Text Overlay</h4>
           </div>
 
           <div class="component-code-snippet"></div>
+
+        <!-- -------------------------- Image ------------------------ -->
+
+        <div class="component-heading">
+          <h2>Responsive Image</h2>
+        </div>
+
+        <div class="component-body">
+          <h4>Image</h4>
+          <div class="component-example-container">
+            <div class="img-container">
+              <img class="img-responsive" src="/images/pottery2.jpeg" alt="responsive-image"/>
+            </div>
+          </div>
+          <div class="component-code-snippet"></div>
+
+          <h4>Rounded Image</h4>
+          <div class="component-example-container">
+            <img class="img-responsive img-rounded" src="/images/cup1.jpg" alt="responsive-image"/>
+          </div>
+          <div class="component-code-snippet"></div>
+
+          <h4>Squared Image</h4>
+          <div class="component-example-container">
+            <img class="img-responsive img-squared" src="/images/cup1.jpg" alt="responsive-image"/>
+          </div>
+          <div class="component-code-snippet"></div>
+
+        </div>
+
+
+        <!-- ------------------------------------------ Input ------------------------------------ -->
+
+        <div class="component-heading">
+          <h2>Input</h2>
+        </div>
+
+        <div class="component-body">
+          <h4>Basic Input</h4>
+          <div class="component-example-container" style="display: block;">
```

Comment:
You are using inline styles for preview purposes, right?

Reply:
Yes, just for preview purposes. But I'll add a class instead.